ProjectSettings/ProjectSettings.asset

```diff
@@ -432,7 +432,8 @@ PlayerSettings:
   webGLUseEmbeddedResources: 0
   webGLUseWasm: 0
   webGLCompressionFormat: 1
-  scriptingDefineSymbols: {}
+  scriptingDefineSymbols:
+    14: 
```

Comment:
True.  I think this started happening recently with the Unity 5.5 update, but wasn't happening on Unity 5.4 HTPs.